docs/guides/redis/sentinel/replacesentinel/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Replace Sentinel
```

Comment:
Wont it be good if we move `Replace Sentinel` out of the sentinel folder, & treat it as same as other opsRequests ?

Reply:
This Ops Request is for a specific Mode. So kept in under that Mode.